easybuild/easyconfigs/r/R-bundle/R-bundles-intel-2015a-R-3.1.3-pbd.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'Bundle'
+
+name = 'R-bundles'
+version = '20150520'
+bundlename = 'pbd'
+rver = '3.1.3'
+versionsuffix = '-R-%s-%s' % (rver, bundlename)
+
+homepage = 'http://www.r-project.org/'
+description = """R is a free software environment for statistical computing and graphics."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+# these are extensions for R
+exts_defaultclass = 'RPackage'
+exts_filter = ("R -q --no-save", "library(%(ext_name)s)")
+
+dependencies = [
+    ('R', rver),
+]
+
+name_tmpl = '%(name)s_%(version)s.tar.gz'
+ext_options = {
+    'source_urls': [
+        'http://cran.r-project.org/src/contrib/Archive/%(name)s',  # package archive
+        'http://cran.r-project.org/src/contrib/',  # current version of packages
+        'http://cran.freestatistics.org/src/contrib',  # mirror alternative for current packages
+    ],
+    'source_tmpl': name_tmpl,
+}
+
+# !! order of packages is important !!
+exts_list = [
+    ('rlecuyer', '0.3-3', ext_options),
+    ('Rmpi', '0.6-5', dict(ext_options.items() + [('patches', ['Rmpi-0.6-5_impi5.patch'])])),
+    ('pbdMPI', '0.2-5', dict(ext_options.items() + [('configureargs', ['--with-mpi-type=INTELMPI'])])),
+    ('pbdSLAP', '0.2-0', dict(ext_options.items() + [('configurevars', ['EXT_LDFLAGS="$LIBSCALAPACK $LIBLAPACK $LIBBLAS"'])])),
+    ('pbdBASE', '0.2-3', dict(ext_options.items() + [('patches', ['Ignore-BI_COMM_GLOBAL-on-intel.patch'])])),
```

Comment:
please change patch name to something like `pbdBase-0.2-3_ignore-BI_COMM_GLOBAL-on-intel.patch